packages/react/src/patterns/blocks/ContentBlockSimple/ContentBlockSimple.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { ContentBlock } from '../../sub-patterns/ContentBlock';
+import { ContentItem } from '../../sub-patterns/ContentItem';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { Image } from '../../../components/Image';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { settings } from 'carbon-components';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * Content Block - Simple pattern
+ *
+ * @param {object} props props object
+ * @param {string} props.copy Content block simple short copy to support the heading
+ * @param {string} props.linkType link type ( simple | jump | card )
+ * @param {object} props.link link object which includes url, link text and target properties.
+ * @param {string} props.heading Content block simple heading
+ * @param {object} props.cta cta object
+ * @param {object} props.contentGroup chilren content element
+ * @param {string} props.mediaType Media type, video or image
+ * @param {object} props.mediaData Data for renderimg the media
+ * @param {Array} props.items Array of objects with data for ContentItems
+ * @returns {*} Content block simple pattern
+ */
+const ContentBlockSimple = ({
+  copy,
+  heading,
+  items,
+  mediaType,
+  mediaData,
+  cta,
+}) => (
+  <ContentBlock
+    heading={heading}
+    copy={copy}
+    items={items}
+    cta={cta}
+    style
+    data-autoid={`${stablePrefix}--content-block-simple`}
+    customClassName={`${prefix}--content-block-simple ${prefix}--col-lg-8`}>
+    <div
+      data-autoid={`${prefix}--content-block-simple__content`}
+      className={`${prefix}--content-block-simple__content`}>
+      {_renderMedia(mediaType, mediaData)}
+      {_renderContent(items)}
+    </div>
+  </ContentBlock>
+);
+
+/**
+ *
+ * @param {string} mediaType Media type, video or image
+ * @param {object} mediaData Data for renderimg the media
+ * @returns {*} JSX Component with the media
+ */
+const _renderMedia = (mediaType, mediaData) => {
+  if (mediaData) {
+    if (mediaType === 'image') {
+      return (
+        <div data-autoid={`${stablePrefix}--content-block-simple__media`}>
+          <Image {...mediaData} />
+        </div>
+      );
+    }
+  }
+};
+
+/**
+ *
+ * @param {Array} items Array of data for ContentItems to be rendered
+ * @returns {*} Array of ContentItem Components
+ */
+const _renderContent = items =>
+  items.map((item, index) => <ContentItem {...item} key={index} />);
```

Comment:
I think we're overcomplicating this.

First off I'm seeing the `heading` being passed through. There are no headings here.

<img src="https://user-images.githubusercontent.com/3793636/74900602-f70f5180-5365-11ea-9f5e-5d53e12e4bfe.png" width="400px" />

Second, the `copy` prop should just be passed down here. With markdown we should be able to format multiple paragraphs here.

```jsx
<ContentBlock heading={heading} cta={cta}>
  <ContentItem copy={copy} />
  <Image {...image} />
</ContentBlock>
```

Reply:
Ok I will change to receive only one `ContentItem` (I think I got my wires crossed on that one), but are you saying that `ContentBlock` should not have `heading` and `cta`? Per the spec:

![image](https://user-images.githubusercontent.com/909118/74940198-15556b80-53bf-11ea-8d3b-ff65059e00d0.png)

Reply:
Only commenting on this because it didn't get resolved. ContentBlock only has heading `1a`, children `2`, and cta `not sure number right now`.